Loader/SymfonyFixturesLoader.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/*
+ * This file is part of the Doctrine Fixtures Bundle.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Doctrine\Bundle\FixturesBundle\Loader;
+
+use Doctrine\Bundle\FixturesBundle\DependencyInjection\CompilerPass\FixturesCompilerPass;
+use Doctrine\Common\DataFixtures\DependentFixtureInterface;
+use Doctrine\Common\DataFixtures\FixtureInterface;
+use Doctrine\Common\DataFixtures\Loader;
+use Symfony\Bridge\Doctrine\DataFixtures\ContainerAwareLoader;
+
+/**
+ * @author Ryan Weaver <ryan@knpuniversity.com>
+ */
+final class SymfonyFixturesLoader extends ContainerAwareLoader
+{
+    public function addFixture(FixtureInterface $fixture)
+    {
+        // see https://github.com/doctrine/data-fixtures/pull/274
+        // this is to give a clear error if you do not have this version
+        if (!method_exists(Loader::class, 'createFixture')) {
```

Comment:
I'd prefer bumping the dependency version for `doctrine/data-fixtures` to `^1.3.0`, but I understand you might not want to do so due to the implied PHP 7.1 requirement for the autoloading bundle version. So, after all, I'm 👍 on this change until it makes sense to bump the PHP requirement for the bundle.

Reply:
Exactly. It's indeed an ugly hack... and I'm sure we'll all feel comfortable removing it at *some* point in the future (hopefully not too far in the future)